test/unit/ui/camera.test.js

```diff
@@ -503,7 +503,7 @@ test('camera', (t) => {
             const camera = createCamera({ zoom: 0 });
             camera.rotateTo(90, { around: [5, 0], duration: 0 });
             t.equal(camera.getBearing(), 90);
-            t.deepEqual(fixedLngLat(camera.getCenter()), fixedLngLat({ lng: 4.999999999999972, lat: 0.000014144426558004852 }));
+            t.deepEqual(fixedLngLat(camera.getCenter()), fixedLngLat({ lng: 4.999999999999972, lat: 0.000002552471840999715 }));
```

Comment:
Quick question — why is value here different? Latitude is close to 0 so isn't supposed to be affected by clamping here...

Reply:
This is caused by a slight change in the `maxValidLatitude` value from `85.05113` to `85.051129`. Should I revert to the older value?